src/frontend/main/map/DorlingCartogram.tsx

```diff
@@ -138,74 +118,38 @@ const Map: React.FunctionComponent<{
       .attr('stroke-width', 1.5)
       .attr('stroke', '#ccd2d5');
     g.selectAll(`.cityCircle`)
-      .data(mapShapeData.features)
+      .data(props.mapShapeData)
       .enter()
       .append('circle')
       .attr('class', 'cityCircle')
-      .attr('cx', (d: {}) => path.centroid(d)[0])
-      .attr('cy', (d: {}) => path.centroid(d)[1])
-      .attr('r', (d: { properties: any }) => {
-        if (d.properties[props.radiusScaleKey] === -1) return props.defaultRadius;
-        return rScale(d.properties[props.radiusScaleKey]);
+      .attr('cx', (d: mapProperties) => d.x)
+      .attr('cy', (d: mapProperties) => d.y)
+      .attr('r', (d: any) => {
+        if (d[props.radiusScaleKey] === -1) return props.defaultRadius;
+        return rScale(d[props.radiusScaleKey]);
       })
       .attr('fill', '#fff')
       .style('cursor', 'pointer')
-      .on('click', (d: {}) => {
+      .on('click', (d: mapProperties) => {
         setActiveCityData(d);
         toggleModal();
       });
-    let tick: () => void = () => {
-      g.selectAll('.cityCircle')
-        .attr('cx', (d: { x: number }) => d.x)
-        .attr('cy', (d: { y: number }) => d.y);
-    };
-    d3.forceSimulation(mapShapeData.features)
-      .force(
-        'x',
-        d3
-          .forceX((d: {}) => path.centroid(d)[0])
-          .strength((d: { properties: { population: number } }) => {
-            if (d.properties.population > 100000) return 10;
-            return 1;
-          }),
-      )
-      .force(
-        'y',
-        d3
-          .forceY((d: {}) => path.centroid(d)[1])
-          .strength((d: { properties: { population: number } }) => {
-            if (d.properties.population > 100000) return 20;
-            return 10;
-          }),
-      )
-      .force(
-        'collide',
-        d3.forceCollide((d: { properties: any }) => {
-          if (d.properties[props.radiusScaleKey] === -1) return props.defaultRadius + 1;
-          return rScale(d.properties[props.radiusScaleKey]) + 1;
-        }),
-      )
-      .alpha(0.1)
-      .on('tick', tick);
 
-    let keyG = mapG
-      .append('g')
-      .attr('class', 'keyG')
-      .attr('transform', `translate(0,-20)`);
+    let keyG = keySVG.append('g').attr('class', 'keyG');
     let colorKey = [...props.colorRange];
     colorKey.reverse().push(props.defaultColor);
-    let colorLegend = ['Ylin 10', '10-20', '20-30', 'Muut', 'Ei tietoa'];
+    let colorLegend = ['Ylin 10', '10-20', '20-10', 'Muut', 'Ei tietoa'];
```

Comment:
This is probably not changed on purpose? (20-30 -> 20-10)

Reply:
Yeah by mistake changed it back :)